newIDE/electron-app/.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 app/www/
 .env
+app/extensions/ReactDeveloperTools
```

Comment:
super nit: configure your text editor to always [finish a file with a new line](https://thoughtbot.com/blog/no-newline-at-end-of-file).